test/js/specs/happy.js

```diff
@@ -208,4 +208,206 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
+          crossDevice.crossDeviceSendLinkBtn.click()
+          crossDevice.verifyCrossDeviceCheckNumberCorrectError(crossDeviceSyncCopy)
+        })
+
+        it('should display error when number is wrong', async () => {
+          driver.get(localhostUrl + `?language=${lang}`)
+          const crossDeviceSyncCopy = documentSelection.copy(lang)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr('123456789')
+          crossDevice.crossDeviceSendLinkBtn.click()
+          crossDevice.verifyCrossDeviceCheckNumberCorrectError(crossDeviceSyncCopy)
+        })
+
+        it('should send sms and navigate to check your mobile screen ', async () => {
+          const crossDeviceCheckYourMobileCopy = crossDeviceCheckYourMobile.copy(lang)
+          driver.get(localhostUrl + `?language=${lang}`)
+          goToCrossDeviceScreen()
+          crossDevice.typeMobileNumebr(testDeviceMobileNumber)
+          crossDevice.crossDeviceSendLinkBtn.click()
+          waitForAlertToAppearAndSendSms()
+          crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTitle(crossDeviceCheckYourMobileCopy)
+        })
+
+        describe('cross device check your mobile screen', async () => {
+
+          it('should verify UI elements of the cross device check your mobile screen', async () => {
+            driver.get(localhostUrl + `?language=${lang}`)
+            const crossDeviceCheckYourMobileCopy = crossDeviceCheckYourMobile.copy(lang)
+            goToCrossDeviceScreen()
+            crossDevice.typeMobileNumebr('07495 023357')
+            crossDevice.crossDeviceSendLinkBtn.click()
+            waitForAlertToAppearAndSendSms()
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTitle(crossDeviceCheckYourMobileCopy)
+            if (lang === 'en') {
+              crossDeviceCheckYourMobile.verifyMobileNumberMessage('We’ve sent a secure link to +447495023357')
+            } else {
+              crossDeviceCheckYourMobile.verifyMobileNumberMessage('Hemos enviado un enlace seguro a +447495023357')
+            }
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileMayTakeFewMinutesMessage(crossDeviceCheckYourMobileCopy)
+            crossDeviceCheckYourMobile.verifycrossDeviceCheckYourMobileTipsHeader(crossDeviceCheckYourMobileCopy)
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTipsFirst(crossDeviceCheckYourMobileCopy)
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileTipsSecond(crossDeviceCheckYourMobileCopy)
+            crossDeviceCheckYourMobile.verifyCrossDeviceCheckYourMobileResendLink(crossDeviceCheckYourMobileCopy)
+          })
+
+          it('should be able to resend sms', async () => {
+            driver.get(localhostUrl)
+            const crossDeviceCheckYourMobileCopy = crossDeviceCheckYourMobile.copy()
+            goToCrossDeviceScreen()
+            crossDevice.typeMobileNumebr('07495 023357')
```

Comment:
Which environment are we using for tests? This won't send a real sms, right?

Reply:
This sends the real sms to the test device we have in the office.

Reply:
This means we send sms every time we run tests or push a commit. That's a lot of money in sms I think. @rfreitas what do you think?

Reply:
After talking to Ricardo about it, the cost should be quite low. Even if we run 10 builds per day and 1 SMS costs 4p, based on the number of tests that will send the SMS, the daily cost will be around £3. So monthly, probably around £50, as we did not always run that many CI jobs on a daily basis.